pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -178,6 +180,17 @@ def create(self, request, path):
         }
         return Response(data, status=http_status.HTTP_202_ACCEPTED)
 
+    def _dispatch_import_collection_task(self, repository, temp_file_path, **kwargs):
```

Comment:
Since the subclasser isn't in control of being able to pass additional kwargs, I don't believe this should get `**kwargs` in its signature.

Reply:
I'm wrong about this because there is one case where you need to pass along these `expected_version`, `expected_namespace`, and `expected_name`. Please disregard.